src/app/page.tsx

```diff
@@ -198,13 +199,18 @@ const Home: React.FC<HomeProps> = () => {
     setConnectModalOpen(true);
   };
 
-  const handleDisconnectBTC = () => {
-    setBTCWallet(undefined);
+  const { clearTermsAccepted } = useAcceptTerms(address, publicKeyNoCoord);
```

Comment:
```suggestion
  const { clearTerms, acceptTerms } = useAcceptTerms(address, publicKeyNoCoord);
```

Reply:
Is `useAcceptTerms` related to the checkbox of accepting terms or related to logging the acceptance of the terms?

Reply:
it now logs the terms acceptance on wallet connect

Reply:
its related to logging the acceptance of the terms